Resources/Public/Javascript/PageView/PageView.js

```diff
@@ -349,11 +358,18 @@ dlfViewer.prototype.displayHighlightWord = function() {
     };
 
     if (hasOwnProperty && this.fulltexts[0] !== undefined && this.fulltexts[0].url !== '' && this.images.length > 0) {
-        var value = urlParams[param],
-            values = value.split(';'),
+        var value = undefined,
             fulltextData = dlfFullTextUtils.fetchFullTextDataFromServer(this.fulltexts[0].url, this.images[0]),
             fulltextDataImageTwo = undefined;
 
+        if(this.highlightWords != null) {
+            value = this.highlightWords;
+        } else {
+            value = urlParams[param];
+        }
+
+        var values = decodeURIComponent(value).split(';'),
```

Comment:
```suggestion
        var values = decodeURIComponent(value).split(';');
```
This causes unfortunately an JavaScript exception which breaks the Pageview.